motionplan/ik/inverseKinematics.go

```diff
@@ -4,30 +4,97 @@ package ik
 
 import (
 	"context"
+	"math"
+	"math/rand"
+	"strings"
 
 	"go.viam.com/rdk/referenceframe"
 )
 
 const (
+	// If limits are infinite, we use this to bound creating random seeds.
+	defaultLimit = 999
```

Comment:
We probably shouldn't let ourselves specify infinite limits and make them define them explicitly.  This feels like a kind of arbitrary number

Reply:
This doesn't limit anything; it's just a starting point for random seeding.

In the case where limits are infinite, and the true solution is 5 million or w/e, then the gradient descent will get us there from here. We just need to pick somewhere to start.

Variable name can probably be more clear for that.